operator/gitops/compute/pipeline-service-manager/role.yaml

```diff
@@ -5,12 +5,106 @@ metadata:
   name: pipeline-service-admin
 rules:
   - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - secrets
+      - serviceaccounts
+      - services
+    verbs:
       - "*"
+  - apiGroups:
+      - apps
```

Comment:
Note - this still maintains "full admin" permissions over all `apps` objects (`Deployment`, `StatefulSet`, etc.). Shouldn't we restrict this to just `Deployment`?